sycl/plugins/esimd_emulator/CMakeLists.txt

```diff
@@ -45,42 +61,23 @@ if (DEFINED CM_LOCAL_SOURCE_DIR)
       BUILD_BYPRODUCTS  ${LIBCM} ${LIBIGFXCMRT_EMU}
     )
   endif()
-else ()
-  if (DEFINED CM_PACKAGE_URL)
-    # Downloading pre-built CM Package
-    file (MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/cm-emu_install)
-    ExternalProject_Add(cm-emu
-      URL               ${CM_PACKAGE_URL}
-      CONFIGURE_COMMAND ""
-      BUILD_COMMAND     ""
-      UPDATE_COMMAND    ""
-      INSTALL_DIR       ${CMAKE_CURRENT_BINARY_DIR}/cm-emu_install
-      INSTALL_COMMAND   ${CMAKE_COMMAND} -E copy_directory ${CMAKE_CURRENT_BINARY_DIR}/cm-emu-prefix/src/cm-emu/ <INSTALL_DIR>
-      BUILD_BYPRODUCTS  ${LIBCM} ${LIBIGFXCMRT_EMU}
-    )
-  else()
-    # Build from CM source tree fetched from github
-    if (MSVC)
-      message(FATAL_ERROR "Online-building of CM_EMU library is not supported under Windows environment")
-    else()
-      # Arguments for online patching to suppress log message from CM_EMU
-      # Replacing CM_EMU's log print-out macro controlled by 'GFX_EMU_WITH_FLAGS_'
-      # with blank space from $CM_EMU_SRC/common/emu_log.h
-      set (replacing_pattern s/{\ ?GFX_EMU_WITH_FLAGS_.*//g)
-      ExternalProject_Add(cm-emu
-        GIT_REPOSITORY    https://github.com/intel/cm-cpu-emulation.git
-        GIT_TAG           f9d167edbcb995e713d5a8
-        BINARY_DIR        ${CMAKE_CURRENT_BINARY_DIR}/cm-emu_build
-        INSTALL_DIR       ${CMAKE_CURRENT_BINARY_DIR}/cm-emu_install
-        PATCH_COMMAND     perl -pi.back -e ${replacing_pattern} ${CMAKE_CURRENT_BINARY_DIR}/cm-emu-prefix/src/cm-emu/common/emu_log.h
-        CMAKE_ARGS        -DLIBVA_INSTALL_PATH=/usr
-                          -D__SYCL_EXPLICIT_SIMD_PLUGIN__=true
-                          -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR>
-        BUILD_BYPRODUCTS  ${LIBCM} ${LIBIGFXCMRT_EMU}
-      )
-    endif()
+else()
+  set(ACTUAL_CM_EMU_PREBUILT_PACKAGE ${DEFAULT_CM_EMU_PREBUILT_PACKAGE})
```

Comment:
I suggest to add a message here so that users can see what `USE_*` variables are available to adjust the emulator build, something like
"Neither USE_DEFAULT_CM_EMU_SOURCE nor USE_LOCAL_CM_EMU_SOURCE is set, using prebuilt libCM from ${ACTUAL_CM_EMU_PREBUILT_PACKAGE}"